### PR TITLE
Blog app - creating a data model

### PR DESCRIPTION
**Pull request by Recillah-Khamala**
### **Blog app - creating a data model**

Hello Code Reviewer,

In this milestone, I have -:
- Created a new app called Kreative Keeper.
- Initialized project with Git.
- Made sure that my project has a Postgres database set up.
- Build my project schema.
  - Created and ran the necessary migration files.
  - Table and column names match the ERD diagram.
  - Foreign keys are included.
  - All columns that are foreign keys have a corresponding index.
- Added an updated README.md file
- Ensured all liters are passing

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,226 @@
-# README
+<a name="readme-top"></a>
+<div align="center">
+  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
+  <img src="Kreative-Keeper.png" alt="logo" width="auto"  height="auto" />
+  <br/>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+  <h3><b>Microverse README Template</b></h3>
```
Comment (andersonlebon) [inline README.md]: # [OPTIONAL]

- [ ] Kindly remove this text to  make your readme file clean

File: README.md [hunk 2]
```diff
@@ -1,24 +1,226 @@
...
+**[Kreative Keeper]** is a fully functional blog website that shows list of posts and empowers readers to interact with them by adding comments and liking posts
 
-* Services (job queues, cache servers, search engines, etc.)
+## 🛠 Built With <a name="built-with"></a>
 
-* Deployment instructions
+### Tech Stack <a name="tech-stack"></a>
 
-* ...
+> Describe the tech stack and include only the relevant sections that apply to your project.
```
Comment (andersonlebon) [inline README.md]: # [OPTIONAL]

- [ ] Kindly remove this text to  make your readme file clean

File: README.md [hunk 3]
```diff
@@ -1,24 +1,226 @@
...
+  <ul>
+    <li><a href="https://www.postgresql.org/">PostgreSQL</a></li>
+  </ul>
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+> Describe between 1-3 key features of the application.
```
Comment (andersonlebon) [inline README.md]: # [OPTIONAL]

- [ ] Kindly remove this text to  make your readme file clean

**Review by andersonlebon — APPROVED** [review]
## Hi @Recillah-Khamala 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

nodding-yes

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Good PR description :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question `@andersonlebon` so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 1 above)
Comment (Recillah-Khamala) [reply README.md]: Thank you. I have removed it.

File: README.md (hunk 2 above)
Comment (Recillah-Khamala) [reply README.md]: Done

File: README.md (hunk 3 above)
Comment (Recillah-Khamala) [reply README.md]: Done

**Pull request merged**